sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormReadingOrder.java

```diff
@@ -3,10 +3,13 @@
 
 package com.azure.ai.formrecognizer.models;
 
+import com.azure.ai.formrecognizer.FormRecognizerServiceVersion;
 import com.azure.core.util.ExpandableStringEnum;
 
 /**
  * Defines values for the order in which recognized text lines are returned.
+ *
+ * This class is introduced since {@link FormRecognizerServiceVersion#V2_1}.
```

Comment:
Use this instead `@since {@link FormRecognizerServiceVersion#V2_1}`. This should be at the bottom of the javadoc.